src/components/views/AgreementForm.js

```diff
@@ -178,7 +178,6 @@ class AgreementForm extends React.Component {
     const { form, initialValues: { id, name } } = this.props;
 
     const hasLoaded = form.getRegisteredFields().length > 0;
-
```

Comment:
Let the code breathe, don't unnecessarily delete empty lines. :)

Reply:
I don't remember doing this, I've added it back in now.